src/main/java/course_project/Game.java

```diff
@@ -0,0 +1,94 @@
+package course_project;
+
+import base.BaseClazz;
+import course_project.services.FieldPrinter;
+import course_project.services.PlayerController;
+import course_project.ship.abstracts.SinglePartShip;
+import course_project.ship.models.Player;
+import course_project.ship.abstracts.TypeShip;
+
+import java.awt.*;
+import java.awt.event.InputEvent;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import static course_project.services.GameService.*;
+
+public class Game extends BaseClazz {
```

Comment:
It's quite difficult to put ships manually, especially because the field does not refresh

Reply:
I got it, you refresh after the type of ships was placed. Smart move

Reply:
Some bug in field rendering 
![image](https://user-images.githubusercontent.com/58872509/134052598-1e33dd8e-bed8-463e-85e7-a645cde5b1c7.png)

Reply:
Also time between turns is quite long, could you please reduce sleep time?